test/v5/stakingAndDAO/kyberDao.js

```diff
@@ -0,0 +1,5696 @@
+                    [1, 2, 3], '0x', {from: campCreator}
+                );
+                assert(false, "throw was expected in line above.")
+            } catch (e) {
+                assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
+            }
+            await daoContract.submitNewCampaign(
+                0, currentBlock + 8, currentBlock + 8 + minCampPeriod, formulaParamsData,
+                [1, 2, 3], '0x', {from: campCreator}
+            );
+        });
+
+        it("Test submit campaign should revert brr camp's already existed", async function() {
+            await deployContracts(30, currentBlock + 20, 4);
+            await daoContract.submitNewCampaign(
+                2, currentBlock + 4, currentBlock + 4 + minCampPeriod, formulaParamsData,
+                [1, 2, 3], '0x', {from: campCreator}
+            );
+            try {
+                await daoContract.submitNewCampaign(
+                    2, currentBlock + 6, currentBlock + 6 + minCampPeriod, formulaParamsData,
+                    [1, 2, 3], '0x', {from: campCreator}
+                );
+                assert(false, "throw was expected in line above.")
+            } catch (e) {
+                assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
+            }
+            await daoContract.submitNewCampaign(
+                0, currentBlock + 8, currentBlock + 8 + minCampPeriod, formulaParamsData,
+                [1, 2, 3], '0x', {from: campCreator}
+            );
+            await daoContract.submitNewCampaign(
+                1, currentBlock + 10, currentBlock + 10 + minCampPeriod, formulaParamsData,
+                [1, 2, 3], '0x', {from: campCreator}
+            );
+            currentBlock = await Helper.getCurrentBlock();
+            // jump to epoch 1
+            await Helper.increaseBlockNumberBySendingEther(accounts[0], accounts[0], startBlock - currentBlock + 1);
+            currentBlock = await Helper.getCurrentBlock();
+            await daoContract.submitNewCampaign(
+                2, currentBlock + 4, currentBlock + 4 + minCampPeriod, formulaParamsData,
+                [1, 2, 3], '0x', {from: campCreator}
+            );
+            try {
+                await daoContract.submitNewCampaign(
+                    2, currentBlock + 6, currentBlock + 6 + minCampPeriod, formulaParamsData,
+                    [1, 2, 3], '0x', {from: campCreator}
+                );
+                assert(false, "throw was expected in line above.")
+            } catch (e) {
+                assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
+            }
+            await daoContract.submitNewCampaign(
+                0, currentBlock + 8, currentBlock + 8 + minCampPeriod, formulaParamsData,
+                [1, 2, 3], '0x', {from: campCreator}
+            );
+        });
+
+        it("Test submit campaign should revert excee
```

Comment:
use open zeppelin expect revert.
or can change the helper to use it.
see how its used in other test files.

Reply:
Sure, saw it from some test files in Katalyst